setup.py

```diff
@@ -20,7 +20,7 @@
     long_description=long_description,
     classifiers=[
         "Development Status :: 4 - Beta",
-        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.10",
```

Comment:
This can have multiple lines like [here](https://github.com/WISDEM/WOMBAT/blob/0bba8c630de9b2e29debd9fa931f5ab922ef3fb4/pyproject.toml#L50-L53). This actually updates the PyPI listing sidebar, so it's generally pretty helpful to list all supported versions.